src/xrpld/app/tx/detail/ConfidentialClawback.cpp

```diff
@@ -98,6 +98,10 @@ ConfidentialClawback::preclaim(PreclaimContext const& ctx)
     if (!sleHolderMPToken->isFieldPresent(sfIssuerEncryptedBalance))
         return tecNO_PERMISSION;
 
+    // Sanity check: claw amount can not exceed confidential outstanding amount
+    if (ctx.tx[sfMPTAmount] > (*sleIssuance)[sfConfidentialOutstandingAmount])
+        return temBAD_AMOUNT;  // LCOV_EXCL_LINE
```

Comment:
this line is reachable. so we shouldn't exclude coverage. It's possible for the issuer to submit the clawback transaction more than COA. Without adding this check, it would be caught during proof verification, which is more expensive. 

Also should be `(*sleIssuance)[~sfConfidentialOutstandingAmount].vallue_or(0) `since COA could still be 0